### DISCUSSION TITLE
Enable maximizing when pressing ctrl+opt+up+up instead of ctrl+opt+return

### DISCUSSION THREAD

**Discussion by charmpitz** [rxhanson/Rectangle#806] [Ideas]
This is something that Cinnamon Linux environment has and that I find very nice. I've tried setting it up but the settings don't seem to support 2x key presses. At least consider finding a way of making this work. Thank you!

**Comment by rxhanson** [comment]
Thanks for the feedback. Been a while since I've used Cinnamon. What's the benefit of this? Is it so you can bind something else to ctrl+opt+up?

**Reply by charmpitz** [reply]
When moving around windows very fast from multiple screens it's much easier to do this only with the arrow keys instead of moving your hand reaching for return key then back to the arrows to do the same for another window. 

I'm using 3 screens and I often need to rearrange everything when connecting my Macbook to the monitors. I have to do the same task for like 3-4 maybe more windows and I find it annoying having to switch my right hand from the arrows to the return. I keep my left hand for cmd + tab / ctrl + option to switch / arrange the windows and the right hand on the arrows to decide which monitor and how to arrange it in there. It would be easier if I didn't have to.

I'm thinking this could work as an improvement. For example when the pressing ctrl + option + up the window will be arranged on the top half but if pressed again now it does nothing. Instead of that it could toggle to fullscreen and back to the top half while pressing the same combination. I hope I explained it well enough 😅, what do you think?
